setup.cfg

```diff
@@ -41,6 +41,9 @@ install_requires =
 [options.packages.find]
 where = src
 
+[options.package_data]
+* = *.tim, *.par
```

Comment:
Do any of our examples use any other kind of file? It'd be preferable if we could specify a directory and everything in it for inclusion, rather than by extension. Not really a problem, though, as long as there's a note somewhere pointing to this as the rule.

Reply:
Pretty sure we can.  I think everything in `src/pint/data` could easily be included.  So maybe a better solution is to make that whole directory installed, and at the same time try to remove all of the separate copies of the par/tim files and update the code (as much as possible) to use this one location.  

A down side is that if we have intentionally bad files it would make them more visible (as you said).  But a fix for that would be a README file (or similar) describing the origin of each file.  In fact that would generally be very useful.

Reply:
I agree that a README would be useful, but I think it's okay for the intentionally-bad files to live in the testing directory somewhere. I suspect that most of the intentionally broken files are created for a single test, or at most a few tests in a single file, and for these I think including them, triple-quoted, and using StringIO might be sufficient (and is there any reason to use a giant, broken, file? most brokenness can probably be tested with a short stub par or tim file). If there are any that are intentionally broken and widely used we probably need to revisit this discussion. 

Another category we need to figure out what to do with is the files for which TEMPO or TEMPO2 output is stored for comparison - do we want to make them and the comparison output part of our distribution, or should that remain internal to PINT testing? (These tests are a headache when we see a discrepancy - who knows why it happens or what causes it... but I agree we have a need for them.)

So I think in the short term I'm inclined to use `pint/data/*` for the things we need for our examples, and keep the testing datadir for weird stuff that doesn't make sense for anyone else to use. Longer-term, I think we should try to shrink the weird testing-only stuff, maybe to zero, but that's part of a longer-term "make our tests less horrible" project.